src/components/pickers/RangeSlider/RangeSlider.stories.tsx

```diff
@@ -0,0 +1,135 @@
+import { useState } from 'react';
+import { Meta, Story } from '@storybook/react';
+
+import { baseProps } from '../../../stories/lists/baseProps';
+import { MULTIPLE_NUMBER_VALUE_ARG } from '../../../stories/FormFieldArgs';
+
+import { CubeRangeSliderProps, RangeSlider } from './RangeSlider';
+
+export default {
+  title: 'Pickers/RangeSlider',
+  component: RangeSlider,
+  args: { id: 'name', width: '200px' },
+  parameters: { controls: { exclude: baseProps } },
+  argTypes: { ...MULTIPLE_NUMBER_VALUE_ARG },
+} as Meta<CubeRangeSliderProps>;
+
+const Template: Story<CubeRangeSliderProps> = (args) => (
+  <RangeSlider {...args} />
+);
+
+export const Single = Template.bind({});
+
+Single.args = {
+  defaultValue: 20,
+  minValue: 0,
+  maxValue: 100,
+  step: 2,
+};
+
+export const Dual = Template.bind({});
+
+Dual.args = {
+  defaultValue: [20, 80],
+  minValue: 0,
+  maxValue: 100,
+  step: 2,
+};
+
+export const Label = Template.bind({});
+
+Label.args = {
+  label: 'Label',
+  defaultValue: [20, 80],
+  minValue: 0,
+  maxValue: 100,
```

Comment:
Replace with `maxValue: 10` and run the story. It will break the Slider. I know it's incorrect data, but it should not break the component.

Reply:
<img width="418" alt="image" src="https://user-images.githubusercontent.com/327209/193114895-b470f2a1-4eb0-4791-9b15-7695c3d4df90.png">